sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/models/package-info.java

```diff
@@ -2,7 +2,5 @@
 // Licensed under the MIT License.
 // Code generated by Microsoft (R) AutoRest Code Generator.
 
-/**
- * Package containing classes for AzureQueueStorage.
- */
+/** Package containing classes for AzureQueueStorage. null. */
```

Comment:
@jianghaolu @srnagar is the `null.` a codegen thing?

Reply:
Most likely. I don't see any literal "null" in the swagger spec.